src/pages/Flow/Automations-Tile.vue

```diff
@@ -0,0 +1,112 @@
+<script>
+import CardTitle from '@/components/Card-Title'
+import AutoCard from '@/pages/Dashboard/Automations/AutomationCard'
+
+export default {
+  components: {
+    CardTitle,
+    AutoCard
+  },
+  props: {
+    flow: {
+      required: true,
+      type: Object
+    }
+  },
+  data() {
+    return {
+      loadingHook: 0,
+      loadCards: false,
+      sortedHooks: []
+    }
+  },
+  watch: {
+    hooks() {
+      this.sortHooks()
+    }
+  },
+  methods: {
+    async sortHooks() {
+      this.loadCards = true
+      const hooks = this.hooks ? [...this.hooks] : []
+      let filtered = hooks.filter(
+        hook => hook.event_tags.flow_group_id === this.flow.flow_group_id
```

Comment:
Flow group ids are returned as an array here.  This means you are missing all state change hooks.